contrib/cmd/user-broker/user-broker.go

```diff
@@ -37,10 +42,39 @@ func init() {
 }
 
 func main() {
+	if err := run(); err != nil && err != context.Canceled && err != context.DeadlineExceeded {
+		glog.Fatalln(err)
```

Comment:
@arschles `glog.Fatalln()` will then exit with non-zero error code which I believe is not the right thing to do - when a process gets a SIGTERM (our signal handler) it should exit and exitting with non-zero means unhappy exit :) But it is a clean shutdown, not a crash or error, so it should be 0, right?

I think in this case we should also be handling `http.ErrServerClosed` and not error out if it happens - because it was us who told the server to stop and it is a part of normal exit.
WDYT?